### ISSUE TITLE
Can't Invite Anyone to Events

### ISSUE THREAD

**Issue by landry314** [AllanWang/Frost-for-Facebook#1703]
When I click 'invite' on any event then choose the recipients, it sometimes does nothing. Has anyone had this issue with frost? Sorry I can't isolate the bug more...

**Comment by AllanWang** [comment]
Sometimes? Is this repeatable for certain events and not others? Does it have to do with the specific event type, or the user you are trying to invite? And does it still cause problems now?

**Comment by landry314** [comment]
Well, now when I select a bunch of people to invite it might invite only a subset of them... I chose about 10 last time and it invited only 5. Before it was inviting none.

It is quite erratic. I really need to open a browser when I want to invite people to events.

**Comment by landry314** [comment]
Yeah, this is still an issue. Sometimes Frost only invites 1 person after I select many to invite. Sometimes none of them. I have no idea how this is working. I have to use a web browser to invite people to events on facebook because Frost doesn't work for this. :-(

**Comment by Perdellian** [comment]
I get this too. I haven't tracked down exactly how/when it happens. Sometimes it'll invite everyone I select, sometimes no one, but most often somewhere in between.

I'd have to double check whether it happens with the mobile site in a webbrowser

**Comment by landry314** [comment]
yes, it is hard to guess how many people it will invite. this bug makes frost impossible to use for inviting people to events.

**Comment by AllanWang** [comment]
If you can, try seeing if it works in your web browser. Even if it's repeatable, I currently have no idea why that would happen. If it's opening something up in another window, I could try disabling that, but aside from that I don't see why it would differ from normal browsers.
